docs/teleport/aks-getting-started.md

```diff
@@ -1,358 +1,222 @@
+* Each ACR used must have Project Teleport enabled.
+* Each ACR must use the *Premium* Tier and be in the same region as your AKS cluster. See [Project Teleport supported regions][teleport-regions].
+* At this time, Project Teleport supports Linux containers on AKS clusters. Windows support is not yet available.
 
-Configure variables unique to your environment. Note the ACR and AKS instances must both be in one of the [teleport supported regions][teleport-regions].
+### Enable Project Teleport on an ACR
 
-```azurecli-interactive
-AKS=myaks
-AKS_RG=${AKS}-rg
-RG_LOCATION=westus2
-K8S_VERSION=1.19.7
-ACR=myacr
-ACR_URL=${ACR}.azurecr.io
-```
-
-## Create an AKS Cluster
-
-At the current moment, Teleport does not yet support managed identity access to teleport expanded layers. Until managed identities are supported, configure the cluster with a service principal.
-
-```azurecli-interactive
-SP_PWD=$(
-  az ad sp create-for-rbac \
-  --skip-assignment \
-  --name ${AKS}-sp 
-  --query password -o tsv)
-
-az aks create \
-    -g ${AKS_RG} \
-    -n ${AKS} \
-    --attach-acr $ACR \
-    --kubernetes-version ${K8S_VERSION} \
-    -l $RG_LOCATION \
-    --service-principal $(az ad sp show \
-        --id http://${AKS}-sp \
-        --query appId \
-        --output tsv) \
-    --client-secret $SP_PWD
-
-az aks get-credentials \
-    -g ${AKS_RG} \
-    -n ${AKS}
-```
+Create a [premium instance][acr-tiers] of Azure Container Registry in one of the [Project Teleport supported regions][teleport-regions].
 
-A standard AKS cluster should now exist. Verify with:
+Sign up for ACR Project Teleport using [the signup form][teleport-signup-form] and provide the resource ID of the ACR. Once enabled, you'll receive an email confirming your ACR has been enabled for Project Teleport.
 
-```azurecli-interactive
-kubectl get nodes
-```
+Once you receive a confirmation email, push an image to a new repository. You may also use `az acr import` to copy an image from another registry.
 
-## Import an image for teleportation
-
-For completeness of this walkthrough, the azure-vote application is used, which includes a 944mb `azure-vote-front:v1` image. To expand the layers, import the images into a teleport enabled registry, in the same region as the AKS cluster.
-
-```azurecli-interactive
-az acr import \
-  --source mcr.microsoft.com/azuredocs/azure-vote-front:v1 \
-  --name $ACR \
-  --image azure-vote-front:v1
-
-az acr import \
-  --source mcr.microsoft.com/oss/bitnami/redis:6.0.8 \
-  --name $ACR \
-  --image redis:6.0.8
-```
-
-## Confirm import and teleport expansion
-
-Confirm the `azure-vote-front` repository is set for teleport expansion:
+To confirm the image has been teleport expanded, use the following command, replacing `<acr-name>` and `<namespace/image>` with your acr and image name.
```

Comment:
Note: This only tells us that the repository is teleport enabled, it doesn't indicate image expansion success. Also some tags could be unexpanded if the repository already existed.